src/main/java/org/fcrepo/importexport/common/Config.java

```diff
@@ -657,4 +667,38 @@ public boolean isIncludeAcls() {
     public void setIncludeAcls(final boolean includeAcls) {
         this.includeAcls = includeAcls;
     }
+
+    /**
+     * @return the number of threads to use, may be null
+     */
+    public Integer getThreadCount() {
+        return threadCount;
+    }
+
+    /**
+     * Set the number of threads to use, must be at least 1
+     *
+     * @param threadCount the number of threads to use
+     */
+    public void setThreadCount(final Integer threadCount) {
+        if (threadCount == null) {
+            this.threadCount = threadCount;
```

Comment:
It seems like this should be 
```suggestion
            this.threadCount = 1;
```

Reply:
It's actually okay for this to be null, and it allows the default value to be calculated here: https://github.com/pwinckles/fcrepo-import-export/blob/FCREPO-3742/src/main/java/org/fcrepo/importexport/exporter/Exporter.java#L792

I think the bug here is that if the thread count is less than 1, it should just ignore the value and use the default, which may or may not be 1. That is to say, I'll ditch the `Math.max()` call.